pulp_2to3_migrate/app/constants.py

```diff
@@ -1,2 +1,35 @@
 # for tasking system to ensure only one migration is run at a time
-PULP_2TO3_MIGRATION_RESOURCE = "pulp_2to3_migration"
\ No newline at end of file
+PULP_2TO3_MIGRATION_RESOURCE = 'pulp_2to3_migration'
+
+# Pulp2 plugins and their content types which can be migrated
+# 'pulp2_plugin': 'pulp2 model class name'
+SUPPORTED_PULP2_PLUGINS = {
+    'iso': ['ISO'],
+    # 'rpm': [
+    #     'Distribution',
+    #     'Drpm',
+    #     'Erratum',
+    #     'Modulemd',
+    #     'ModulemdDefaults',
+    #     'PackageCategory',
+    #     'PackageEnvironment',
+    #     'PackageGroup',
+    #     'PackageLangpacks',
+    #     'Rpm',
+    #     'Srpm',
+    #     'YumRepoMetadataFile'],
+    # 'docker': [
+    #     'Blob',
+    #     'Image',
+    #     'Manifest',
+    #     'ManifestList',
+    #     'Tag'],
+}
+
+# Mapping Pulp 2 content type to Pulp 3 plugin and content type
+# 'pulp2 content type id' -> ('pulp3 plugin', 'pulp3 content type')
+PULP_2TO3_TYPE_MAP = {
```

Comment:
Not used for now.
@dkliban , let me know if it's useful for the other task you wrote or I can remove it.

Reply:
It is useful. I need to look up model name from a string.